doc/start/os-recommendations.rst

```diff
@@ -39,6 +39,25 @@ platforms.  Generally speaking, there is very little dependence on
 specific distributions aside from the kernel and system initialization
 package (i.e., sysvinit, upstart, systemd).
 
+Octopus (15.2.z)
+-----------------
+
++----------+----------+--------------------+--------------+---------+------------+
+| Distro   | Release  | Code Name          | Kernel       | Notes   | Testing    |
++==========+==========+====================+==============+=========+============+
+| CentOS   | 8        | N/A                | linux-4.18   |         | B, I, C    |
++----------+----------+--------------------+--------------+---------+------------+
+| Centos   | 7        | N/A                | linux-3.10.0 | 3       | B, I,      |
```

Comment:
please drop "3" from the notes column, we don't test with btfs anymore.

Reply:
Thanks, I've clarified the btrfs situation on Octopus